I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 14, in script
    image auak = Transform("auak.png", scale=1.3)
  File "game/script.rpy", line 14, in <module>
    image auak = Transform("auak.png", scale=1.3)
Exception: Unknown transform property: 'scale'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 14, in script
    image auak = Transform("auak.png", scale=1.3)
  File "C:\Users\Usuario\Documents\renpy\renpy-8.3.7-sdk\renpy\ast.py", line 933, in execute
    img = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\Usuario\Documents\renpy\renpy-8.3.7-sdk\renpy\python.py", line 1211, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 14, in <module>
    image auak = Transform("auak.png", scale=1.3)
  File "C:\Users\Usuario\Documents\renpy\renpy-8.3.7-sdk\renpy\display\transform.py", line 745, in __init__
    raise Exception("Unknown transform property: %r" % k)
Exception: Unknown transform property: 'scale'

Windows-10-10.0.22621 AMD64
Ren'Py 8.3.7.25031702
Kaua 1.0
Thu May  8 18:24:31 2025
